### DISCUSSION TITLE
Welcome to gesticulator Discussions!

### DISCUSSION THREAD

**Discussion by Svito-zar** [Svito-zar/gesticulator#20] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Elbastu** [comment]
Dear Mr Taras,
Could you provide a colab version of the code ?
thank you!

**Reply by Svito-zar** [reply]
Dear @Allaoui-Mohamed-El-Bastu, thank you for your question. 
I have never used Google Colab myself before and have no time to learn it now, so colab version will not come soon, unfortunately.
On top of that, the dataset we use requires users to first sign a license before access. Not sure if that is compatible with putting the project on Google colab....

**Reply by Elbastu** [reply]
I requested data access a week ago. I'm still waiting for authorization. 
Thank you
